ballerina/tests/resources/documents/field_object_astronauts.graphql

```diff
@@ -0,0 +1,10 @@
+query{
+    astronauts{
+        missions{
+            id
+            designation
+            startDate
+            endDate
+        }
+    }
+}
```

Comment:
Check all the occurrences. There should be an empty line at the end of each file.

```suggestion
}

```